.travis.yml

```diff
@@ -1,2 +1,13 @@
-language: python
-script: pytest tests/
+matrix:
+  include:
+    - language: python
+      before_script:
+        - cd python
+      script:
+        - pytest tests
+    - language: r
```

Comment:
I think we should take out `R` coverage from TravisCI first. We can include it in a separate PR later. Let's focus on merging this